.travis.yml

```diff
@@ -0,0 +1,47 @@
+language: python
+
+python:
+  - "2.7"
+
+env:
+#  - PYFITS=false INSTALL_TYPE=develop TEST=submodule
+  - PYFITS=true INSTALL_TYPE=develop TEST=submodule
+#  - PYFITS=false INSTALL_TYPE=install TEST=package
+#  - PYFITS=true INSTALL_TYPE=install TEST=package
+#  - PYFITS=true INSTALL_TYPE=install TEST=smoke
+
+before_install:
+  - sudo apt-get update -qq
+  - sudo apt-get install -qq build-essential gfortran flex bison
+  - wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh
+  - chmod +x miniconda.sh
+  - ./miniconda.sh -b
+  - export PATH=/home/travis/miniconda/bin:$PATH
+  - conda update --yes conda
+  - conda install --yes python=$TRAVIS_PYTHON_VERSION pip numpy matplotlib
+  - conda config --add channels https://conda.binstar.org/cxc
+  - pip install -r test_requirements.txt
+  - if [ ${TEST} == package ];
+     then pip install ./sherpa-test-data;
+     git submodule deinit -f .;
+    fi
+  - if [ ${TEST} == smoke ];
+     then git submodule deinit -f .;
+    fi
+
+install:
+  - python setup.py $INSTALL_TYPE
+
+script:
+  - if [ ${PYFITS} == true ]; then conda install --yes pyfits; fi
```

Comment:
Put this line in the `before_install` section where the other dependencies are installed?

I think pyfits is already deprecated or it will be in the coming years, so better to install astropy and test against `astropy.io.fits`? (Maybe in addition to `pyfits` as a separate build if that's important to CIAO?)